optaplanner-quarkus-bom/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-parent</artifactId>
+    <version>8.10.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>optaplanner-quarkus-bom</artifactId>
+  <packaging>pom</packaging>
+
+  <properties>
+    <!-- SNAPSHOT and KIE dependency versions -->
+    <version.org.kie.kogito>2.0.0-SNAPSHOT</version.org.kie.kogito>
+
+    <version.com.thoughtworks.xstream>1.4.17</version.com.thoughtworks.xstream>
+    <version.io.quarkus>2.0.1.Final</version.io.quarkus>
+    <version.org.apache.commons.math3>3.6.1</version.org.apache.commons.math3>
+    <version.org.freemarker>2.3.31</version.org.freemarker>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-bom</artifactId>
+        <version>${project.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-bom</artifactId>
```

Comment:
This is the trick to avoid aligning with versions coming from Quarkus over and over. We reuse the `quarkus-bom` and use this new `optaplanner-quarkus-bom` as a parent in our `optaplanner-build-parent`.